recipes/xqilla/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(XercesC REQUIRED)
+find_package(xqilla REQUIRED)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} XercesC::XercesC xqilla::xqilla)
```

Isnt xqilla pulling this requirement in? why do we need to link here as well?

```suggestion
target_link_libraries(${PROJECT_NAME} xqilla::xqilla)
```

---

XercesC find_package and linkage removed.